### ISSUE TITLE
A11y: Unable to login with just keyboard shortcuts at the password field

### ISSUE THREAD

**Issue by lyqht** [GeeksHacking/flagit-docs#38]
Usually you can do "Enter" at the last input or "Cmd+Shift+Enter" to submit a whole form. 
However, users can only login by clicking the button

![CleanShot 2024-07-19 at 22 33 01@2x](https://github.com/user-attachments/assets/d68bf97c-b4bb-4615-9dab-132ea99a6adf)
